pkg/exporter/msg.go

```diff
@@ -30,7 +41,7 @@ func CreateIPFIXMsg(set entities.Set, obsDomainID uint32, seqNumber uint32, expo
 	msgLen := entities.MsgHeaderLength + set.GetSetLength()
 	if msgLen > entities.MaxSocketMsgSize {
```

Comment:
Should we use expProc.maxMsgSize here? This should be the same case when input.MaxMsgSize=0, but I'm kind of confused about what if user set their own MaxMsgSize?

Reply:
I chose to keep this function as is, as it is a freestanding function (also used by tests) and not an exporter method. This function only enforces the default upper bound. The user-provided upper-bound is enforced in the exporter before calling this function.

There would be no real harm in removing this check altogether but I thought I would keep it just for cases where the function is called by tests.